shell/platform/darwin/ios/framework/Source/SemanticsObject+UIFocusSystem.mm

```diff
@@ -87,16 +145,99 @@ - (CGRect)frame {
   //
   // This method is only supposed to return items within the given
```

Comment:
just a note that I have run into weird bug before because i returned child something outside of the rect of accessibilityContainer. I am not sure if it applies here.

Reply:
Yeah I can try adding culling here but I'll have to test it separately (and it seems to be working fine w/o the culling).